### PR TITLE
Sync 9.x to main

### PR DESCRIPTION
**Pull request by LightGuard**
Hopefully there isn't anything we need to clean up.

### CODE REVIEW

File: introduction/release-notes/release-notes-9-0-0.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
+= BAMOE, 9.0.0, release notes
+include::../../partials/attributes.adoc[]
+
+Highlights, new features, removed components, and differences between {PRODUCT} {VERSION_OLD} and {VERSION}:
```
Comment (thiagoelg) [inline introduction/release-notes/release-notes-9-0-0.adoc:4]:
```suggestion
Highlights, new features, removed components, and differences between {PRODUCT} {VERSION_80} and {VERSION_900}:
```

File: partials/attributes.adoc [hunk 2]
```diff
@@ -10,7 +10,7 @@
 :PRODUCT_PAM: IBM Process Automation Manager Open Edition
 :PRODUCT_PAM_SHORT: PAMOE
 :OPENSHIFT: OpenShift
-:VERSION: 9.0.0
+:VERSION: 9.0.1
 :VERSION_901: 9.0.1
 :VERSION_90X : 9.0.x
```
Comment (thiagoelg) [inline partials/attributes.adoc:15]:
```suggestion
:VERSION: 9.0.1
:VERSION_900: 9.0.0
:VERSION_901: 9.0.1
:VERSION_90X : 9.0.x
```

**Review by thiagoelg — CHANGES_REQUESTED** [review]
Just a couple of changes regarding the `VERSION_OLD` and `VERSION_900` variables.

Also, we can change `KIE_TOOLS_COMMUNITY_VERSION` to 0.30.4.

Thanks for the changes @LightGuard!

**Comment by LightGuard** [comment]
Suggestions accepted, thanks for the catch, @thiagoelg !

**Review by thiagoelg — APPROVED** [review]

**Comment by LightGuard** [comment]
@porcelli @baldimir @thiagoelg @eduardocerqueira please review so we get get this merge to main and sync the work we did on `9.x` on `main`.

**Comment by LightGuard** [comment]
@porcelli @baldimir @thiagoelg please review. (@thiagoelg I got your change request in)

**Review by porcelli — APPROVED** [review]
lgtm

**Comment by LightGuard** [comment]
Perfect! Thank you everyone.

**Pull request merged**